### ISSUE TITLE
Additional properties are not allowed ('debug_id' was unexpected)

### ISSUE THREAD

**Issue by Kotevode** [getsentry/sentry-cli#980]
### Environment

How do you use Sentry?
Self-hosted@9.1.2

Which SDK and version?
sentry-cli@1.66.0 through fastlane-plugin-sentry@1.8.1

### Steps to Reproduce

1. Build and archive iOS app with bitcode disabled and DWARF with dSYM file debug information format
2. Run sentry_upload_dsym

### Expected Result

Processed and uploaded symbols

### Actual Result

```
[20:42:10]: --------------------------------
[20:42:10]: --- Step: sentry_upload_dsym ---
[20:42:10]: --------------------------------
[20:42:11]: sentry-cli 1.66.0 installed!
[20:42:11]: Starting sentry-cli...
[20:42:15]: > Found 1 debug information file
[20:42:15]: > Prepared debug information file for upload
[20:42:16]: error: API request failed
[20:42:16]: caused by: sentry reported an error: Additional properties are not allowed ('debug_id' was unexpected) (http status: 400)
```

I assume the latest sentry-cli is incompatible with our self-hosted instance therefore the question is what version of sentry-cli should I use?

**Comment by flub** [comment]
It seems https://github.com/getsentry/sentry/commit/296a423b3653df7b81c9e500fbfd8bd941b7f1d5 introduced this requirement accidentally in sentry gated on the `ChunkedUploadCapability::Pdbs` capability, but accidentally made this schema change not gated on the capability.

The corresponding commit in sentry-cli is https://github.com/getsentry/sentry-cli/commit/9e25d168f7d068f1721377507173d220cf88e882.  From those two commits (in)compatible releases could be figured out I guess.

I'm not sure if fixing the sentry side is worth it, given that this is an ancient release.  sentry-cli could potentially be fixed to not emit the `ChunkedDifRequest::debug_id` field if the `Pdbs` capability is missing, but again this is an old release and the fix is probably harder than it sounds as getting access to the capabilities at the right point in sentry-cli might not be easy.

**Comment by flub** [comment]
That's not the entire compatibility story I think, while that does explain the protocol bug it was not triggered because sentry-cli used to be less eager to supply a debug_id.  https://github.com/getsentry/sentry-cli/blob/e224e0ebf5120b21273b156c105e78e7ac26fc51/src/utils/dif_upload.rs#L122-L125 is probably responsible for being more eager in supplying a `debug_id`, previously I think it just supplied `None` all the time.

That would make https://github.com/getsentry/sentry-cli/commit/5392a66c5cc183fdfc699f56c37875546d7189f3 the commit from which sentry-cli might have become incompatible.  This could be much easier to fix (though remain somewhat brittle).

**Comment by chreck** [comment]
I had the same issue and I could solve it by using these versions for my react-native project.

**package.json**
```
"dependencies": {
...
        "@sentry/react-native": "1.9.0",
        "@sentry/cli": "1.61.0",
...
```

**Comment by makoni** [comment]
Why is this till broken in 1.69.1?

The last one that works is 1.64.2

**Comment by alinradut** [comment]
I can confirm this is still broken on 1.70.0.

**Comment by ValentinDenis** [comment]
Still broken on 1.71.0

**Comment by kamilogorek** [comment]
Should be fixed in `sentry-cli@1.72.1`.
